service/auth.go

```diff
@@ -128,6 +170,37 @@ func (s *Service) unregisterClient(w http.ResponseWriter, r *http.Request) {
 	}
 
 	s.log.Debug("unregistered client", mlog.String("clientID", clientID))
+	data.code = http.StatusOK
+}
+
+func (s *Service) loginClient(w http.ResponseWriter, r *http.Request) {
+	if r.Method != http.MethodPost {
+		http.NotFound(w, r)
+		return
+	}
+
+	data := &httpData{
+		reqData: map[string]string{},
+		resData: map[string]string{},
+	}
+	defer s.httpAudit("loginClient", data, w, r)
+
+	if err := json.NewDecoder(r.Body).Decode(&data.reqData); err != nil {
+		data.err = err.Error()
+		data.code = http.StatusBadRequest
+		return
+	}
+
+	clientID := data.reqData["clientID"]
+	authKey := data.reqData["authKey"]
+	bearerToken, err := s.auth.Login(clientID, authKey)
+	if err != nil {
+		data.err = err.Error()
+		data.code = http.StatusBadRequest
+		return
+	}
 
+	s.log.Debug("logged in client", mlog.String("clientID", clientID), mlog.String("bearerToken", bearerToken))
```

Comment:
We probably don't want to log authentication tokens. The client ID is sufficient.